Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/DeviceIssuesPage/OpenXDAIssuesPage.tsx

```diff
@@ -35,25 +35,29 @@ import moment from 'moment';
 
 function OpenXDAIssuesPage(props: { Meter: OpenXDA.Types.Meter }) {
     const [data, setData] = React.useState<SC.OpenXDADailyStatistic[]>([]);
-    const [sortField, setSortField] = React.useState<keyof SC.OpenXDADailyStatistic>('Date');
+    const [sortField, setSortField] = React.useState<keyof SC.OpenXDADailyStatistic>('LastSuccessfulFileProcessed');
     const [ascending, setAscending] = React.useState<boolean>(false);
-
+    
     React.useEffect(() => {
         let handle = $.ajax({
             type: "GET",
-            url: `${homePath}api/SystemCenter/Statistics/OpenXDA/${props.Meter.AssetKey}`,
+            url: `${homePath}api/SystemCenter/Statistics/OpenXDA/${props.Meter.AssetKey}/${sortField}/${ascending ? '1' : '0'}`,
             contentType: "application/json; charset=utf-8",
             dataType: 'json',
             cache: false,
             async: true
-        }) as JQuery.jqXHR<SC.OpenXDADailyStatistic[]>;
-        handle.done(d => setData(orderBy(d, [sortField], [ascending ? "asc" : "desc"])))
+        });
+        handle.done(d => setData(JSON.parse(d.toString())))
```

Comment:
I am assuming the `JSON.parse` is necessary because `d` is a `string`? If that is the case Tying for `handle` should be `JQuery.jqXHR<string>`